src/fields/field_diagnostics.f90

```diff
@@ -189,6 +204,30 @@ end function get_horizontal_enstrophy
 
         !::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
+        ! maximum horizontal enstrophy, i.e. sqrt(xi^2 + eta^2)
+        function get_max_horizontal_enstrophy() result(hemax)
+            double precision :: hemax
+
+            hemax = maxval(dsqrt(vor(:, :, :, 1) ** 2 + vor(:, :, :, 2) ** 2))
+
+            call mpi_blocking_reduce(hemax, MPI_MAX, world)
+
+        end function get_max_horizontal_enstrophy
+
+        !::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+        ! maximum horizontal enstrophy, i.e. sqrt(xi^2 + eta^2)
```

Comment:
I would remove this - we never need this because the horizontal enstrophy is non-negative.  The max value is all we need

Reply:
I removed the calculation of the minimum value. Did you mean this to remove?

Reply:
Yes, anything to do with minimum horizontal enstrophy should be removed.  No minimum enstrophy of any form should be diagnosed.

Reply:
Ok. I did so.